src/adagrams.js

```diff
@@ -1,15 +1,104 @@
+const availableLetters = {
+  'A': [9, 1],
+  'B': [2,3], 
+  'C': [2, 3], 
+  'D': [4, 2], 
+  'E': [12, 1], 
+  'F': [2, 4], 
+  'G': [3, 2], 
+  'H': [2, 4], 
+  'I': [9, 1], 
+  'J': [1, 8], 
+  'K': [1, 5], 
+  'L': [4, 1], 
+  'M': [2, 3], 
+  'N': [6, 1], 
+  'O': [8, 1], 
+  'P': [2,3], 
+  'Q': [1,10], 
+  'R': [6,1], 
+  'S': [4,1], 
+  'T': [6,1], 
+  'U': [4,1], 
+  'V': [2,4], 
+  'W': [2,4], 
+  'X': [1,8], 
+  'Y': [2,4], 
+  'Z': [1,10]
+}
+
 export const drawLetters = () => {
```

Comment:
I would use the spread operator to make a copy of the letterPool
use a for loop to draw 10 letters
use math.floor to make the random draw
push the letter to letterHand
return letterHand